src/invidious/helpers/errors.cr

```diff
@@ -26,22 +26,53 @@ def error_template_helper(env : HTTP::Server::Context, locale : String?, status_
   if exception.is_a?(InfoException)
     return error_template_helper(env, locale, status_code, exception.message || "")
   end
+
   env.response.content_type = "text/html"
   env.response.status_code = status_code
-  issue_template = %(Title: `#{exception.message} (#{exception.class})`)
+
+  issue_title = "#{exception.message} (#{exception.class})"
+
+  issue_template = %(Title: `#{issue_title}`)
   issue_template += %(\nDate: `#{Time::Format::ISO_8601_DATE_TIME.format(Time.utc)}`)
   issue_template += %(\nRoute: `#{env.request.resource}`)
   issue_template += %(\nVersion: `#{SOFTWARE["version"]} @ #{SOFTWARE["branch"]}`)
   # issue_template += github_details("Preferences", env.get("preferences").as(Preferences).to_pretty_json)
   issue_template += github_details("Backtrace", exception.inspect_with_backtrace)
+
+  # URLs for the error message below
+  url_faq = "https://github.com/iv-org/documentation/blob/master/FAQ.md"
+  url_search_issues = "https://github.com/iv-org/invidious/issues"
+
+  url_switch = "https://redirect.invidious.io" + env.request.resource
+
+  url_new_issue = "https://github.com/iv-org/invidious/issues/new"
+  url_new_issue += "?labels=bug&template=bug_report.md&title="
+  url_new_issue += URI.encode_www_form("[Bug] " + issue_title)
+
   error_message = <<-END_HTML
-    Looks like you've found a bug in Invidious. Please open a new issue
-    <a href="https://github.com/iv-org/invidious/issues">on GitHub</a>
-    and include the following text in your message:
-    <pre style="padding: 20px; background: rgba(0, 0, 0, 0.12345);">#{issue_template}</pre>
+    <div class="error_message">
+      <h2>#{translate(locale, "crash_page_you_found_a_bug")}</h2>
+      <br/><br/>
+
+      <p><b>#{translate(locale, "crash_page_before_reporting")}</b></p>
+      <ul>
+        <li><a href="#{env.request.resource}">#{translate(locale, "crash_page_refresh")}</a></li>
+        <li><a href="#{url_switch}">#{translate(locale, "crash_page_switch_instance")}</a></li>
+        <li>#{translate(locale, "crash_page_read_the_faq", url_faq)}</li>
+        <li>#{translate(locale, "crash_page_search_issue", url_search_issues)}</li>
+      </ul>
+
+      <br/>
+      <p>#{translate(locale, "crash_page_report_issue", url_new_issue)}</p>
+
+      <!-- TODO: Add a "copy to clipboard" button -->
```

Comment:
It's not possible without JavaScript... so it's not great.

Reply:
Yeah, the button would be created from JS